SPEC.md

```diff
@@ -43,3 +44,17 @@
 ```
 
 The version number must be valid [semver](https://semver.org). See the [npm docs](https://docs.npmjs.com/misc/semver) for more info.
+
+### `main`
+
+Module ID that is the primary entry point to a package - that is when consumer requests the package i.e. with `require('my-package')` the resolution algorithm should check this field and redirect the request to the specified value. **(Default)** `index.js`
+
+#### Rules
+
+* Value should be a path relative to the root of a package
+* Value gets resolved with node's resolution algorithm, described [here](http://nodejs.cn/doc/nodejs/modules.html#modules_all_together)
```

Comment:
It might be good to make a whole section about this and just link to it, also from other fields that will get described in the future because I believe all of those fields should be resolved using the same algorithm (can be file, can be a directory with index.js or package.json, can be extension-less etc). 

Other than that other fields should only describe why they are here, what kind of "redirect" or "alias" files should they contain, their shape (i.e. browser can be an object) and relation between each others (like priorities between them in certain contexts etc)